### ISSUE TITLE
Preact build not copying over manifest.json and assets

### ISSUE THREAD

**Issue by cjoy** [preactjs/preact-cli#910]
**Do you want to request a _feature_ or report a _bug_?**
Bug

**What is the current behaviour?**
When using  ```preact build --no-prerender```, the `manifest.json` and `assets/*` files get replaced with the default template ones. 

**If the current behaviour is a bug, please provide the steps to reproduce.**
I'm using the typescript template. I only added an additional build flag in the package.json file.

**What is the expected behaviour?**
I'm assumed my `manifest.json` file would be copied over.

**Please mention other relevant information.**
I'm using the latest Preact X.

**Comment by prateekbh** [comment]
is `manifest.json` missing in `build/` after running preact build?

**Comment by prateekbh** [comment]
also what version of preact-cli are you on?

**Comment by mrchrisjoy** [comment]
@prateekbh  I can see the original `manifest.json` file (the one that was there when the project was generated). 

I'm using `3.0.0-next.19`

**Comment by LorenzoZaccagnini** [comment]
Same issue, I have always the default manifest.json and assets in the build, preact 10.1.0 and preact-cli 3.0.0-rc.6

**Comment by LorenzoZaccagnini** [comment]
This is my temporary fix in package.json
`  "scripts": {
    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
    "build": "preact build --template src/template.html && npm run copy_manifest",
    "copy_manifest": "cp manifest.json build/manifest.json && cp assets/icon.png build/assets/icon.png",
    "serve": "preact build && preact serve",
    "dev": "preact watch --template src/template.html",
    "lint": "eslint src"
  },`

Basically I copy manifest.json and the icon

**Comment by devalexandre** [comment]
for copy you can , move assets and manifest inner src

**Comment by prateekbh** [comment]
just checked. seems to be working on rc17, please re-open if this persist
